debian/postinst

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+cat << HEREDOC > /etc/ood/profile
+#!/bin/bash
+
+export GEM_PATH="/opt/ood/gems:$GEM_PATH"
+HEREDOC
+
+cat << HEREDOC > /etc/sudoers.d/ood
+Defaults:www-data !requiretty, !authenticate
+Defaults:www-data env_keep += "NGINX_STAGE_* OOD_*"
+www-data ALL=(ALL) NOPASSWD: /opt/ood/nginx_stage/sbin/nginx_stage
+HEREDOC
+
+chmod 440 /etc/sudoers.d/ood
+
+mkdir -p /var/lib/ondemand-nginx
+mkdir -p /var/lob/ondemand-nginx
+mkdir -p /var/tmp/ondemand-nginx
+mkdir -p /var/run/ondemand-nginx
+
+mkidir -p /var/lib/ondemand-nginx/config/{puns,apps}
+mkidir -p /var/lib/ondemand-nginx/config/apps/{sys,dev,usr}
+touch /var/lib/ondemand-nginx/config/apps/sys/dashboard.conf
```

Comment:
This looks like the steps needed to avoid dashboard initialization, this is essentially what the RPM does that avoids initialization.  For the RPM we actually package the touched files rather than touch in a post script.